packages/itwinui-css/backstop/scenarios/input.js

```diff
@@ -13,4 +13,20 @@ module.exports = [
   scenario('Type Without Label', {
     selectors: ['#demo-no-label'],
   }),
+  scenario('State hover in default', {
+    actions: [hover('#test-input')],
+    selectors: ['#demo-input'],
```

Comment:
Can you add `viewports: [{ width: 800, height: 600 }],` to all of these new tests?

Reply:
Added

Reply:
Can you add these to textarea as well?

Reply:
Good catch! Fixed